Wireframe/css/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+header {
+    background-color: #f4f4f4;
+    padding: 20px;
+    text-align: center;
+}
+
+h1 {
+    margin: 0;
+}
+
+main {
+    padding: 20px;
+}
+
+article {
+    border: 1px solid #ccc;
+    margin: 10px;
+    padding: 15px;
+}
+
+article:first-child {
+    display: block;
+    background-color: #e0f7fa;
+    font-weight: bold;
+}
+
+article:not(:first-child) {
```

Comment:
Do you need this not? What would happen is you wrote  a general rule for article and then a few differences for first-child?

Reply:
not(:first-child) rule is used to style all articles except the first one, allowing them to display side by side. This keeps the first article distinct while arranging the others neatly on the page based on the provided wireframe task.